stuffioAPI/stuffioAPI/Controllers/ChatsController.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using stuffioAPI.Models;
+
+namespace stuffioAPI.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/Chats")]
+    [EnableCors("MyPolicy")]
+    public class ChatsController : Controller
+    {
+        private readonly ChatContext _context;
+
+        public ChatsController(ChatContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Chats
+        [HttpGet]
+        public IEnumerable<Chat> GetChats()
+        {
+            return _context.Chats;
+        }
+
+        // GET: api/Chats/GetBySender/2
+        [HttpGet("GetChatsForUser/{id}")]
+        public async Task<IActionResult> GetChatsForUser([FromRoute] long id)
+        {
+            if (!ModelState.IsValid)

```

Comment:
Checking `ModelState` does nothing in a `GET` action, because the request sent no model to validate.

Remove.

Reply:
Noted. Removed